homeassistant/components/binary_sensor/nest.py

```diff
@@ -65,7 +64,7 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
     # Add all available binary sensors if no Nest binary sensor config is set
     if discovery_info == {}:
-        conditions = _VALID_BINARY_SENSOR_TYPES
+        conditions = _VALID_BINARY_SENSOR_TYPES.keys()
```

Comment:
This change isn't needed.

Reply:
I changed `_VALID_BINARY_SENSOR_TYPES` to dict from list, so need `keys()` here.

Reply:
No, iterating a dict iterates the keys by default.